src/packages/commands/rat_command.py

```diff
@@ -76,10 +81,46 @@ class NameCollisionException(CommandException):
                 f"Prefixless rule {getattr(command_fun, '__name__', '')} matching {ctx.words[0]} "
                 f"found.")
 
+    if ctx.words_eol[0].startswith("Incoming Client:"):
+        command_fun = handle_ratmama_announcement
+
     if command_fun:
         return await command_fun(ctx, *extra_args)
 
-    logger.debug(f"Ignoring message '{ctx.words_eol[0]}'. Not a command or rule.")
+    # neither a rule nor a command, possibly a fact
+    result = False
+    if ctx.prefixed:
+        result = await handle_fact(ctx)
+    if not result:
+        logger.debug(f"Ignoring message '{ctx.words_eol[0]}'. Not a command or rule.")
+
+
+async def handle_fact(context: Context):
+    """
+    Handles potential facts
+    """
+    logger.trace("entering fact handler")
+
+    raw = context.words[0]
+    logger.debug("checking {!r} for facts...", raw)
+    if "-" in raw:
+        fact, lang, *_ = raw.split("-")
+    else:
+        fact = raw
+        lang = 'en'
+    try:
+        # don't do anything if the fact doesn't exist
+        if not await context.bot.fact_manager.exists(fact, lang):
+            logger.debug("no such fact name={!r} lang={!r}", fact, lang)
+            return False
+
+        logger.debug("fact exists, retrieving and returning!")
+        fact = await context.bot.fact_manager.find(fact, lang)
+        await context.reply(fact.message)
+        return True
+    except psycopg2.Error:
```

Comment:
Can we use a more specific error handler here? I feel this is too broad.

Reply:
Nope! not without having some way of asking the FM if it has a connection or not.

Reply:
You can.  if the pool is None or in an error state there's no active connection.  I believe there's also a 'connected' property I added.

Reply:
We discussed this, and decided there was a better way - by adding a method to check if the current connection exists.  This should also be considered non-blocking and out of scope.

Reply:
For the sake of keeping the drillsqueak beta alive, we will leave this change in for now. 
As per our discussion, @shatteredbeam will provide better methods for achieving this end without such an ugly exception clause in a later PR.

Reply:
SPARK-164